api/managementpb/dbaas/kubernetes.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package dbaas.v1beta1;
+
+option go_package = "dbaasv1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+
+// TODO move it to parent directory / package once API is v1-stable
+
+// KubeAuth represents Kubernetes / kubectl authentication and authorization information.
+message KubeAuth {
+  // Kubeconfig file content.
+  string kubeconfig = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+// KubernetesCluster contains public info about kubernetes cluster.
+message KubernetesCluster {
+  // Kubernetes cluster name.
+  string kubernetes_cluster_name = 1;
+}
+
+message ListKubernetesClustersRequest {}
+
+message ListKubernetesClustersResponse {
+  // Kubernetes clusters.
+  repeated KubernetesCluster kubernetes_clusters = 1;
+}
+
+message RegisterKubernetesClusterRequest {
+  // Kubernetes cluster name.
+  string kubernetes_cluster_name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Kubernetes auth.
+  KubeAuth kube_auth = 2 [
+    (validator.field) = {
+      msg_exists: true
+    }
+  ];
+}
+
+message RegisterKubernetesClusterResponse {}
+
+message UnregisterKubernetesClusterRequest {
+  // Kubernetes cluster name.
+  string kubernetes_cluster_name = 1 [
```

Comment:
@AlekSi why do we use kubernetes_cluster_name instead of kubernetes_cluster_id like in other places where we remove things?

Reply:
It is k8s resource name, not PMM Inventory object